packages/falso/src/lib/text.ts

```diff
@@ -1,105 +1,15 @@
-import { rand } from './core';
+export function text({ charCount = 10 } = {}): string | void {
```

Comment:
I propose to change the parameter part to a more standard typescript usage:  `text({charCount: number} = { charCount: 10 })`